docs/source/authentication.rst

```diff
@@ -134,58 +134,69 @@ The following is a list of auth backends for the community edition to help get t
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| use_ssl                     | no       | false           | Use LDAPS to connect                                                                                                                     |
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| use_tls                     | no       | false           | Start TLS on LDAP to connect                                                                                                             |
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| cacert                      | no       | None            | Path to the CA cert used to validate certificate                                                                                         |
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| id_attr                     | no       | uid             | Field name of the user ID attribute                                                                                                      |
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| account_pattern             | no       | {id_attr}=      | LDAP subtree pattern to match user. The user's username is escaped and interpolated into this string (see example).                      |
+|                             |          | {{username}}    |                                                                                                                                          | 
++-----------------------------+----------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| group_pattern               | no       | (|(&            | LDAP subtree pattern for user groups. Both user_dn and username are escaped and then interpolated into this string (see example.)        |                      
+|                             |          | (objectClass=*) |                                                                                                                                          | 
```

Comment:
I think the `.rst` is not happy here because of the `*`.
Quick googling on the error message suggests this solution: https://stackoverflow.com/a/30454883/4533625

```suggestion
|                             |          | (objectClass=\*) |                                                                                                                                          | 
```

Let's see if that works

Reply:
I think i tried before escaping the  but it did not helped.

Reply:
Looks like you found the solution, good stuff :+1: